src/variable.cpp

```diff
@@ -128,6 +128,30 @@ double eval_dbl(const variable &var, const std::unordered_map<std::string, doubl
     }
 }
 
+long double eval_ldbl(const variable &var, const std::unordered_map<std::string, long double> &map,
+                      const std::vector<long double> &)
+{
+    if (auto it = map.find(var.name()); it != map.end()) {
+        return it->second;
+    } else {
+        throw std::invalid_argument("Cannot evaluate the variable '" + var.name()
```

Comment:
just a minor thing, but would you mind using the ``_format`` syntax here and perhaps in the ``dbl`` implementation as well? Trying to slowly use ``_format`` everywhere.